### ISSUE TITLE
ERROR [iOS]: use of undeclared identifier 'kGAMSimulatorID'; did you mean 'kGADSimulatorID'

### ISSUE THREAD

**Issue by Sheleya** [ammarahm-ed/react-native-admob-native-ads#254]
node_modules/react-native-admob-native-ads/ios/RNAdmobNativeAdsManager.m:43:96: error: use of undeclared identifier 'kGAMSimulatorID'; did you mean 'kGADSimulatorID'?
        NSArray *testDevices = RNAdMobProcessTestDevices([config valueForKey:@"testDeviceIds"],kGAMSimulatorID);
                                                                                               ^~~~~~~~~~~~~~~
                                                                                               kGADSimulatorID
In module 'GoogleMobileAds' imported from /node_modules/react-native-admob-native-ads/ios/RNAdmobNativeAdsManager.m:4:
/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:15:30: note: 'kGADSimulatorID' declared here
GAD_EXTERN const id _Nonnull kGADSimulatorID;

"react-native-admob": "^2.0.0-beta.6",
"react-native-admob-native-ads": "^0.5.0",

**Comment by Kazu1247** [comment]
Same here

**Comment by ppalbuquerque** [comment]
I had the same issue here.

I solved this issue in my project by using a specific SDK version, instead of the latest version.

```
pod 'Google-Mobile-Ads-SDK', ' 8.11.0' 
pod 'GoogleMobileAdsMediationFacebook', '6.5.1.0'
```

**Comment by RenatoSerra22** [comment]
> I had the same issue here.
> 
> I solved this issue in my project by using a specific SDK version, instead of the latest version.
> 
> ```
> pod 'Google-Mobile-Ads-SDK', ' 8.11.0' 
> pod 'GoogleMobileAdsMediationFacebook', '6.5.1.0'
> ```

life saver! was stuck with this for hours

**Comment by harisraza123** [comment]
> I had the same issue here.
> 
> I solved this issue in my project by using a specific SDK version, instead of the latest version.
> 
> ```
> pod 'Google-Mobile-Ads-SDK', ' 8.11.0' 
> pod 'GoogleMobileAdsMediationFacebook', '6.5.1.0'
> ```

Hi, I got the following error when trying to install pods with
```
pod 'GoogleMobileAdsMediationFacebook', '6.5.1.0'
```
<img width="730" alt="Screenshot 2022-02-23 at 5 36 47 AM" src="https://user-images.githubusercontent.com/69842208/155243501-9e946f97-9684-4dac-ac9d-ecd3f7b7d506.png">

**Comment by CharlyAnd** [comment]
I'm getting the same error, even after specifying the SDK version as people have done abov
<img width="265" alt="Screenshot 2022-04-27 at 13 39 39" src="https://user-images.githubusercontent.com/94613508/165519963-d2f6e7ac-edd6-443c-95dc-a9b495b63192.png">
e

**Comment by ibrahimkanber** [comment]
I solved this issue via 
I was able to fix this problem by patching

**Comment by kyimoemin** [comment]
@ibrahimkanber patch works for me, I already have Google-Mobile-Ads-SDK - 9.3.0 installed. so just add
pod 'GoogleMobileAdsMediationFacebook'. I believe it won't last if I reinstall npm so plz fix asap contributors, it's not much work.

**Comment by ammarahm-ed** [comment]
Issue is fixed in v0.6.0

**Comment by prox2** [comment]
> Issue is fixed in v0.6.0

Hi, i have installed latest version and issue still exists.
